src/tests/evil/unposix/test_evil_unposix_strings.c

```diff
@@ -0,0 +1,46 @@
+// #include "src\lib\evil\unposix\strings.h";
+#include "strings.h"
+#include <stdio.h>
+
+int equals_strings(){
+    char *str1,*str2;
+    str1 = "fOo";
+    str2 = "FoO";
+    
+    return strcasecmp(str1,str2);
+}
```

Comment:
Why not...
```suggestion
int equals_strings()
{
   return strcasecmp("fOo", "FoO");
}
```

It even has a plus of fitting EFL coding conventions :)

Reply:
loll, i'll change it

Reply:
According to EFL style..:

```suggestion
int 
equals_strings()
{
    return strcasecmp("fOo", "FoO");
}
```